src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/Cohort.cs

```diff
@@ -119,6 +120,36 @@ public DraftApprenticeship AddDraftApprenticeship(DraftApprenticeshipDetails dra
             return draftApprenticeship;
         }
 
+        public void SendToOtherParty(Party modifyingParty, string message, UserInfo userInfo, DateTime now)
+        {
+            CheckModifierIsEmployerOrProvider(modifyingParty);
+            EnsureModifierIsAllowedToModifyDraftApprenticeship(modifyingParty);
```

Comment:
It looks like the name of this method is misleading. Could we make it about the Cohort, and not DraftApprenticeship? Perhaps we could work "Party" into the name as well instead of `Modifier` and `Check` instead of `Ensure` so it matches the other? (Personally, I like calling this type of method `Assert...` but some don't seem to like the word Assert outside of a Test.) Also, it looks like this method just calls `ModifierIsAllowedToEdit` and possibly throws. As there are no other references to `ModifierIsAllowedToEdit`, could we refactor it out?